src/lightning_lite/strategies/deepspeed.py

```diff
@@ -186,8 +186,6 @@ def __init__(
 
             logging_batch_size_per_gpu: Config used in DeepSpeed to calculate verbose timing for logging
                 on a per sample per second basis (only displayed if logging=logging.INFO).
-                If set to "auto", the plugin tries to infer this from
-                the train DataLoader's BatchSampler, else defaults to 1.
```

Comment:
We can't infer this automatically at the moment. See note about limitation below